script.js

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
nice job organizing everything into an Object. How could you abstract out some of shared functionality to use prototypes and constructors?